rewrite-java-test/src/test/java/org/openrewrite/java/RemoveUnusedImportsTest.java

```diff
@@ -1072,6 +1072,68 @@ public static void main(String[] args) {
         );
     }
 
+    @Issue("https://github.com/openrewrite/rewrite/issues/3275")
+    @Test
+    void doesNotRemoveReferencedClassesBeingUsedAsParameters() {
+        rewriteRun(
+          java(
+            """
+              package com.Source.$;
+
+              public class A {
+                  public static final short SHORT1 = (short)1;
+
+                  public short getShort1() {
+                    return SHORT1;
+                  }
+              }
+              """
+          ),
+          java(
+            """
+              package com.test;
+
+              import com.Source.$.A;
+
+              class Test {
+                  void f(A classA) {
+                    classA.getShort1();
+                  }
+              }
+              """
+          )
+        );
+
```

Comment:
I'm wondering if this should be two tests; we don't usually see two invocations of `rewriteRun` in a single test method.

Reply:
I am happy to split them out